1001-no-hardcoded-colors.md

```diff
@@ -0,0 +1,18 @@
+---
+category: Design
+state: draft
+created: 2014-10-22
+updated: 2014-10-22
+---
+
+# Clarity design colors
+
+Design teams consuming Clarity Components in figma.
+
+## Guidance
+
+- Design teams **Must** use color tokens provided trough Clarity Components.
```

Comment:
Design teams **Must** use color tokens from Clarity components to keep designs consistent and on-brand.

Reply:
moved to PR #119